frontend/src/app/agentrail.css

```
@@ -0,0 +1,851 @@
+     that has to be visible without being looked for. */
+  /* One length, and never a percentage: a size that resolves against the card's
+     width in one axis and the viewport in the other would give the ball two
+     different diameters. 232px is the width the card can give it, and 24vh is
+     what a short window can, whichever is smaller. */
+  --coreSize: min(232px, 24vh);
+  --coreGlass: calc(var(--coreSize) - 6px);
+  /* The viewport cap is for a hero standing in a content column; the rail's own
+     width and height are the only caps this one needs. */
+  --coreCap: 100%;
+  --coreHalo: 0.4;
+  flex: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)